changelog.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
+
+## 19.11.2020
+
+* Stopped supporting hotel ids in URL.
```

Comment:
```suggestion
* Stopped support for hotel/enterprise ids in URL and in the configuration of a javascript client
```

Reply:
Ah sorry, it was never mentioned. But I would say we can be better safe than sorry here.